settings_yaml.gemspec

```diff
@@ -16,8 +16,8 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'hashie', '~> 3.4'
-  spec.add_runtime_dependency 'activesupport-core-ext', '~> 4.0'
+  spec.add_dependency 'hashie', '~> 3.4'
```

Comment:
Yes, it seems, that `add_runtime_dependency` is the preferred way:
http://guides.rubygems.org/specification-reference/#add_runtime_dependency